### PR TITLE
Add a way to manually re-trigger deploys with GITHASH

### PR DESCRIPTION
**Pull request by jdotpz**
When a job fails on the terraform step, it's shameful to have to start all over again and do AMI build and version build.  Further, it's a PITA to manually copy in the payload to Jenkins.

Thus, this makes us able to setup jobs in Jenkins, which you can manually run, supply an github commit hash, and it will then replace the environments AMI's and swap out nodes.

### CODE REVIEW

File: bin/lib/create_rpm.sh [hunk 1]
```diff
@@ -25,7 +25,7 @@ function create_rpm() {
     # Find the rpm file we created
     NEWRPM=`ls -lart /data/socorro/socorro*.rpm|awk '{print $9}'`
     # Get the version of that socorro rpm so we log it nicely.
-    NEWSOCORROVERSION=`ls /home/centos/org.mozilla.crash-stats.packages-public/x86_64/socorro-*.rpm|tail -n1|sed 's/\// /g'|sed 's/\./ /g'|awk '{print $8}'`
+    NEWSOCORROVERSION=`ls -lart /home/centos/org.mozilla.crash-stats.packages-public/x86_64/socorro-*.rpm|tail -n1|sed 's/\// /g'|sed 's/\./ /g'|awk '{print $8}'`
```
Comment (rhelmer) [inline bin/lib/create_rpm.sh]: why do you need `sed 's/\// /g'|sed 's/\./ /g'` here, just replacing `/` and `.` with a space?

I think `tr` is more readable for this: `tr ['.','/'] ' '`

**Comment by rhelmer** [comment]
r+, take or leave my `tr` vs `sed` comment.

File: bin/lib/create_rpm.sh (hunk 1 above)
Comment (jdotpz) [inline bin/lib/create_rpm.sh]: I'll actually clean that up everywhere it exists in the scripts, but push onward with merging this.

File: bin/deploy-socorro.sh [hunk 2]
```diff
@@ -29,7 +35,7 @@ RANDOM_STRING="$RANDOM-$RANDOM"
 function show_usage() {
         echo " =================== ";echo " ";
         echo " Syntax "
-        echo " ./deploy-socorro.sh $env (stage or prod)"
+        echo " ./deploy-socorro.sh $env [manual](optional) [$githash-to-deploy](optional)"
```
Comment (phrawzty) [inline bin/deploy-socorro.sh]: Please remove the `(optional)` strings - in *nix, one uses chevrons to denote `<required args>` and brackets to denote `[optional ones]`.

Also, as per my previous recommendation in a different PR, any variable you put in double quotes will be interpolated, which means that Bash will attempt display the values both `$env` and `$githash-to-deploy`, which is almost certainly not the intended outcome here. Your options: escape the `$` signs, use single-quotes for this string, or don't use `$` at all.

Recommend:
```bash
echo "USAGE: $0 <env> [manual] [githash-to-deploy]"
echo "Where <env> is the environment, [manual] is <true|false>, and [githash-to-deploy] is a <git short hash>."
```

**Pull request merged**